src/protocol.h

```diff
@@ -412,14 +412,22 @@ class CInv
     std::string ToString() const;
 
     // Single-message helper methods
-    bool IsMsgTx()        const { return type == MSG_TX; }
-    bool IsMsgWtx()       const { return type == MSG_WTX; }
+    bool IsUndefined() const { return type == UNDEFINED; }
+    bool IsMsgTx() const { return type == MSG_TX; }
+    bool IsMsgBlk() const { return type == MSG_BLOCK; }
```

Comment:
[continuing conversation from [another PR](https://github.com/bitcoin/bitcoin/pull/19611#discussion_r463552637)]

nit: `s/Blk/Block/`

```sh
$ git grep -i blk src |wc -l
     108
$ git grep -i block src |wc -l
   11014
```

Reply:
I prefer to pass on this for these reasons:
(a) to maintain a consistent naming style in CInv (`tx`, `blk`)
(b) `block` would make the methods names longer
(c) `blk` is an established abbreviation: in the codebase, block data files called `blk*.dat`, etc.
(d) the shorter `blk` seems more consistent with the abbreviated naming style in use in recent PRs like #19569